lapack.go

```diff
@@ -136,3 +136,23 @@ const (
 	InitZ   Comp = 'I'
 	UpdateZ Comp = 'V'
 )
+
+// EigVecSide specifies what eigenvectors will be computed.
+type EigVecSide byte
```

Comment:
If we were to modify blas, I would make it
```
const (
    Left Side = 141 + iota
    Right
    Both   // <- add this
)
```
instead of mixing it with `Uplo.All`.